modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -283,50 +252,112 @@
+/*
+	ENGINEERING GUARD DATUMS
+*/
+/datum/job/engineering_guard
+	title = JOB_ENGINEERING_GUARD
+	description = "Monitor the supermatter, keep an eye on atmospherics, make sure everyone is wearing Proper Protective Equipment."
+	department_head = list(JOB_CHIEF_ENGINEER)
+	faction = FACTION_STATION
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = SUPERVISOR_CE
+	selection_color = "#ffeeee"
+	exp_granted_type = EXP_TYPE_CREW
+	config_tag = "ENGINEERING_GUARD"
+
+	outfit = /datum/outfit/job/engineering_guard
+	plasmaman_outfit = /datum/outfit/plasmaman/engineering
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_ENG
+
+	display_order = JOB_DISPLAY_ORDER_ENGINEER_GUARD
+	bounty_types = CIV_JOB_ENG
+	departments_list = list(
+		/datum/job_department/engineering,
+		)
+
+	family_heirlooms = list(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec/engineering)
+
+	mail_goodies = list(
+		/obj/item/food/donut/caramel = 10,
+		/obj/item/food/donut/matcha = 10,
+		/obj/item/food/donut/blumpkin = 5,
+		/obj/item/clothing/mask/whistle = 5,
+		/obj/item/melee/baton/security/boomerang/loaded = 1
+	)
+	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
+
+/datum/outfit/job/engineering_guard
+	name = "Engineering Guard"
+	jobtype = /datum/job/engineering_guard
+
+	belt = /obj/item/modular_computer/tablet/pda/engineering
+	ears = /obj/item/radio/headset/headset_eng
+	shoes = /obj/item/clothing/shoes/workboots
+	uniform = /obj/item/clothing/under/rank/security/officer/blueshirt/skyrat/engineering_guard
+	head =  /obj/item/clothing/head/helmet/blueshirt/skyrat/guard
+	suit = /obj/item/clothing/suit/armor/vest/blueshirt/skyrat/engineering_guard
+
+	backpack = /obj/item/storage/backpack/industrial
+	satchel = /obj/item/storage/backpack/satchel/eng
+	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
+	box = /obj/item/storage/box/survival/engineer
+
+	id_trim = /datum/id_trim/job/engineering_guard
 
+/datum/id_trim/job/engineering_guard
+	assignment = "Engineering Guard"
+	trim_icon = 'modular_skyrat/master_files/icons/obj/card.dmi'
+	trim_state = "trim_engiguard"
+	orbit_icon = "shield-heart"
+	sechud_icon_state = SECHUD_ENGINEERING_GUARD
+	extra_access = list(ACCESS_SECURITY, ACCESS_BRIG_ENTRANCE, ACCESS_ENGINEERING, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS, ACCESS_MECH_ENGINE, ACCESS_AUX_BASE,
+					ACCESS_EXTERNAL_AIRLOCKS, ACCESS_CONSTRUCTION, ACCESS_ATMOSPHERICS, ACCESS_TCOMMS, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_BRIG_ENTRANCE, ACCESS_ENGINEERING, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS, ACCESS_MECH_ENGINE, ACCESS_AUX_BASE,
+					ACCESS_EXTERNAL_AIRLOCKS, ACCESS_CONSTRUCTION, ACCESS_ATMOSPHERICS, ACCESS_TCOMMS, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
```

Comment:
It can be done in a later PR if it's too much trouble, but having these lists be multiline would make them a lot easier to read through than how they are at the moment.